src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,82 @@
+package duke;
+
+import java.util.ArrayList;
+
+
+/**
+ * TaskList class is an extension of the java Array List class as it provides functionalities
+ * that are unique for duke.
+ */
+public class TaskList extends ArrayList<Task> {
+
+    public TaskList() {
+        super();
+    }
+
+    /**
+     * Function that prints all the tasks currently in the TaskList.
+     *
+     */
+
```

Comment:
> No blank line between the documentation block and the method/class

This blank line is not needed according to [cs2103t](https://se-education.org/guides/conventions/java/index.html#comments)

This appears at the following as well:
```
     */

    public Task markTask(int index) {
```
```
     */

    public Task unmarkTask(int index) {
```
```
     */

    public Task delete(int index) {
```